src/core/Akka.API.Tests/CoreAPISpec.ApproveRemote.approved.txt

```diff
@@ -28,6 +28,7 @@ namespace Akka.Remote
     public class AkkaProtocolSettings
     {
         public AkkaProtocolSettings(Akka.Configuration.Config config) { }
+        public System.TimeSpan HandshakeTimeout { get; }
```

Comment:
Added new handshake timeout settings to API, per https://github.com/akka/akka/pull/18542